src/main/scala/tilelink/Monitor.scala

```diff
@@ -527,14 +527,16 @@ class TLMonitor(args: TLMonitorArgs, monitorDir: MonitorDirection = MonitorDirec
     }
   }
 
+  @chiselName
```

Comment:
Did this do anything? Seeing as you  still had to add all the suggestName below?